Paintroid/src/main/java/org/catrobat/paintroid/ui/zoomwindow/DefaultZoomWindowController.kt

```diff
@@ -29,32 +30,30 @@ class DefaultZoomWindowController
     (val activity: MainActivity,
     val layerModel: LayerContracts.Model,
     val workspace: Workspace,
-    val toolReference: ToolReference) :
+    val toolReference: ToolReference,
+    val sharedPreferences: UserPreferences) :
     ZoomWindowController {
 
     private val canvasRect = Rect()
     private val checkeredPattern = Paint()
     private val framePaint = Paint()
 
-    private val windowHeight =
-        activity.resources.getDimensionPixelSize(R.dimen.pocketpaint_zoom_window_height)
-    private val windowWidth =
-        activity.resources.getDimensionPixelSize(R.dimen.pocketpaint_zoom_window_width)
+    private val windowSideDimen = activity.resources.getDimensionPixelSize(R.dimen.pocketpaint_zoom_window_height)
 
```

Comment:
You might need to use correct window dimensions `windowSideDimen = activity.resources.getDimensionPixelSize(R.dimen.pocketpaint_zoom_window_height)` is used as width later on

Reply:
I will take a look at it, thanks!